### ISSUE TITLE
Microbe editor organelle list is displayed incorrectly

### ISSUE THREAD

**Issue by KnightosMaster** [Revolutionary-Games/Thrive#451]
When going into the microbe editor the organelle list does not begin at the top, and I can't scroll down to the bottom of it.
![thrive_008](https://cloud.githubusercontent.com/assets/655652/17599022/7a86b9d2-5ffd-11e6-8f77-fe722bce667e.png)

I'm using release 0.3.2 on Arch Linux.

**Comment by hhyyrylainen** [comment]
I have the same issue. 
I'll make sure that it isn't some old file causing this issue. Other than that it might be a linux specific issue.

**Comment by hhyyrylainen** [comment]
On slack jjonj confirmed that this bug occurs on windows, too. And seems to be caused by running in windowed mode.

**Comment by Untrustedlife** [comment]
Its happening on windows for me aswell

**Comment by hhyyrylainen** [comment]
@Untrustedlife Can you check if it works correctly in fullscreen mode? That seems to be what causes this, and not using windows or linux.

**Comment by Untrustedlife** [comment]
Were there any changes made to the microbe editor gui code since the 3.2 release?

**Comment by pgelinas** [comment]
Yes, in the SVN assets there was a change related to compound panel. I'm looking into it.

**Comment by pgelinas** [comment]
Alright, I think this was related to the change in CEGUI version (windows and linux were not using the same version for 0.3.2). Once we updated the mingw setup for the newer CEGUI, we saw this problem appearing on Windows too. Problem was in the XML layout, I guess the newer CEGUI doesn't handle "upward" offsets (UDim(1.0, -808)) the same way it did. I've changed the way we position the Organelle button to the sane way: starts at 0.0 than go down with offset.

Update assets and rebuid.
